examples/simple/main.cpp

```diff
@@ -104,6 +104,21 @@ int main() {
         auto now_s = duration_cast<std::chrono::seconds>(now.time_since_epoch());
         std::cout << "Predicted median EoD: " << eod_median << " s (T- "
                   << (eod_median-now_s.count())<< " s)" << std::endl;
+
+        // The event state or SOC is also part of the ProgEvent object returned by .getEvents().
+        // Each element of the UData vaector corresponds to a prediction at some save point.
+        // Current SOC corresponds to the first element.
+        UData currentSOC = eod_event.getState()[0];
+        // Similar to RUL, SOC contains uncertainty due to the Monte Carlo prediction method.
+        if (currentSOC.uncertainty() != UType::Samples) {
+            std::cerr << "Unexpected uncertainty type for SOC prediction" << std::endl;
+            return std::exit(1);
```

Comment:
```suggestion
            return 1;
```

I don't think std::exit is a return type. It should be ```return 1;``` or ```exit(1);```. On mac this returns the error 'cannot initialize return object of type int from rvalue of type void'

Reply:
oops, yes. That was an error in translation on my part.